### PR TITLE
Templating Console.php, Issue#12007

### PR DESCRIPTION
**Pull request by 007durgesh219**
Before submitting pull request, please check that every commit:
- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them

Signed-off-by: Durgesh 007durgesh219@gmail.com

### CODE REVIEW

File: libraries/Console.php [hunk 1]
```diff
@@ -77,25 +78,12 @@ public function disable()
...
     {
-        $output = '';
         $cfgBookmark = PMA_Bookmark_getParams();
         if ($cfgBookmark) {
-
-            $tpl_bookmark_actions
-                = '<span class="action collapse">' . __('Collapse') . '</span> '
-                . '<span class="action expand">' . __('Expand') . '</span> '
-                . '<span class="action requery">' . __('Requery') . '</span> '
-                . '<span class="action edit_bookmark">' . __('Edit') . '</span> '
```
Comment (zixtor) [inline libraries/Console.php]: 'Edit' option is missing from the template.

**Comment by zixtor** [comment]
The divs '.toolbar.collapsed', .message.collapsed, are repeated all over with different data, so it can be extracted into a separate template.

**Comment by 007durgesh219** [comment]
OK, Thanks. I will update my PR with all changes shortly

**Comment by 007durgesh219** [comment]
@zixtor This is just for review not complete. I tried to convert repeated divs into separate templates. Please let me know if I am going in right direction. I will do rest later.

File: templates/console/query_action.phtml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        if (isset($content)): ?>
+        <span class="<?= $content[0] ?>">
+            <?= $content[1] ; ?>
+            <?php if(isset($content['extraSpan'])): ?>
+                : <span> <?= $content['extraSpan'] ?> </span>
+            <?php endif ; ?>
+        </span>
+    <?php endif ;
+        endforeach ; ?>
+    </span>
```
Comment (zixtor) [inline templates/console/query_action.phtml]: Extra closing tag here.

File: templates/console/query_action.phtml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class="<?= $parentDivClasses ; ?>"
```
Comment (zixtor) [inline templates/console/query_action.phtml]: Missing '>'

**Comment by zixtor** [comment]
@007durgesh219 yes, it is better now. Please fix a couple of things as in the comments and it should be ready to merge.

**Comment by 007durgesh219** [comment]
Thanks @zixtor I will correct those mistakes.  It was just to ensure that I am going in right direction thats why I didn't paid attention on those small mistakes.

**Comment by zixtor** [comment]
It's okay :) and it's indeed in the right direction.

**Comment by 007durgesh219** [comment]
I've updated PR. Please Review it.

**Comment by zixtor** [comment]
Thanks!

**Pull request merged**